cachi2/core/package_managers/pip.py

```diff
@@ -761,16 +772,16 @@ def _find_setup_call(
         if self._is_setup_call(root_node):
             return root_node, []
 
-        for name, field in ast.iter_fields(root_node):
+        for name, value in ast.iter_fields(root_node):
             # Field is a node
-            if isinstance(field, ast.AST):
-                setup_call, setup_path = self._find_setup_call(field)
+            if isinstance(value, ast.AST):
```

Comment:
I noticed that with the variable name change, it no longers match the comment above.